ui/hooks/metamask-notifications/useCreateSession.test.ts

```diff
@@ -0,0 +1,54 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+import * as redux from 'react-redux';
+import * as actions from '../../store/actions';
+import { useCreateSession } from './useCreateSession';
+
+jest.mock('../../store/actions', () => ({
+  signIn: jest.fn(),
+  signOut: jest.fn(),
+  enableMetaMetrics: jest.fn(),
+  disableMetaMetrics: jest.fn(),
+  performSignIn: jest.fn(),
+  disableProfileSyncing: jest.fn(),
+}));
+
+jest.mock('react-redux', () => {
+  const actualRedux = jest.requireActual('react-redux');
+  return {
+    ...actualRedux,
+    useSelector: jest.fn(),
+    useDispatch: jest.fn(() =>
+      jest.fn((action) => {
+        if (typeof action === 'function') {
+          return action();
+        }
+        return action;
+      }),
+    ),
+  };
+});
+
+describe('useCreateSession', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should initialize correctly', () => {
+    (redux.useSelector as jest.Mock)
+      .mockReturnValueOnce(false)
```

Comment:
Hmm, I'm not convinced on these tests.
I don't know if I fully like the idea of mocking redux, but I can understand some difficult parts if there are communications through the controller bridge.

FUTURE - lets tidy up these tests.

Reply:
IIWIW, lets improve tests in a future PR.

Reply:
https://consensyssoftware.atlassian.net/browse/NOTIFY-660